providers/default.rb

```diff
@@ -80,9 +80,9 @@ def update_rubygems
       raise 'cannot find omnibus install' unless ::File.exist?(gem_bin)
       source = "--clear-sources --source #{new_resource.rubygems_url}"
       if Gem::Requirement.new(nodoc_rubygems_versions).satisfied_by?(rubygems_version)
-        shell_out!("#{gem_bin} update --system --no-document #{source}")
+        shell_out!("#{gem_bin} update --system #{target_version} --no-document #{source}")
```

Comment:
What happens on a Chef install that has 3.0 installed? Do we attempt to downgrade it?

Reply:
this code branch doesn't execute then because you don't fail the check for >= 2.6.11

Reply:
yep